modules/hub/README.md

```diff
@@ -0,0 +1,5 @@
+# Overview
+The hub should only do work that is necessary for the deployment of Workers,
```

Comment:
Does this need to be in the README? The README holds end-user docs. The end-user doesn't care about this.